components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,248 @@
+            driveId: this.drive,
+            includeItemsFromAllDrives: true,
+            supportsAllDrives: true,
+          });
+        }
+        return results;
+      },
+    },
+    uploadType: {
+      type: "string",
+      label: "Upload Type",
+      description: `The type of upload request to the /upload URI. If you are uploading data (using an /upload URI), this field is required. If you are creating a metadata-only file, this field is not required. 
+        media - Simple upload. Upload the media only, without any metadata.
+        multipart - Multipart upload. Upload both the media and its metadata, in a single request.
+        resumable - Resumable upload. Upload the file in a resumable fashion, using a series of at least two requests where the first request includes the metadata.
+      `,
+      options: [
+        {
+          label: "Media (upload the media only, without any metadata)",
+          value: "media",
+        },
+        {
+          label: "Multipart (upload both the media and its metadata, in a single request)",
+          value: "multipart",
+        },
+        {
+          label: "Resumable (Upload the file in a resumable fashion, using a series of requests)",
+          value: "resumable",
+        },
+      ],
+    },
+    fileUrl: {
+      type: "string",
+      label: "File URL",
+      description:
+        "The URL of the file you want to upload to Google Drive. Must specify either File URL or File Path.",
+      optional: true,
+    },
+    filePath: {
+      type: "string",
+      label: "File Path",
+      description:
+        "The path to the file, e.g. /tmp/myFile.csv . Must specify either File URL or File Path.",
+      optional: true,
+    },
+    ignoreDefaultVisibility: {
+      type: "boolean",
+      label: "Ignore Default Visibility",
+      description: "Whether to ignore the domain's default visibility settings for the created file. Domain administrators can choose to make all uploaded files visible to the domain by default; this parameter bypasses that behavior for the request. Permissions are still inherited from parent folders.",
+      default: false,
+    },
+    includePermissionsForView: {
+      type: "string",
+      label: "Include Permissions For View",
+      description:
+        "Specifies which additional view's permissions to include in the response. Only 'published' is supported.",
+      optional: true,
+      options: [
+        "published",
+      ],
+    },
+    keepRevisionForever: {
+      type: "boolean",
+      label: "Keep Revision Forever",
+      description:
+        "Whether to set the 'keepForever' field in the new head revision. This is only applicable to files with binary content in Google Drive. Only 200 revisions for the file can be kept forever. If the limit is reached, try deleting pinned revisions.",
+      default: false,
+    },
+    ocrLanguage: {
+      type: "string",
+      label: "OCR Language",
+      description:
```

Comment:
Need options array for this prop

Reply:
Created file language-codes.js for prop options